File: ibm/src/dotnet/Latency/Latency.cs
using System;
using Newtonsoft.Json.Linq;

namespace Latency
{
    public class LatencyDotnet
    {
        public JObject Main(JObject args)
        {
            JObject message = new JObject();
            message.Add("success", new JValue(true));
            JObject payload = new JObject();
            payload.Add("test", new JValue("latency test"));
            message.Add("payload", payload);

            return (message);
        }
    }
 }

File: ibm/src/go/latency/latency.go
package main

func Main(params map[string]interface{}) map[string]interface{} {

    msg := map[string]interface{}{}
    msg["success"] = true
    msg["payload"] = map[string]interface{}{}
    msg["payload"].(map[string]interface{})["test"] = "latency test"

    return msg

}